src/swap-helpers.ts

```diff
@@ -267,3 +278,23 @@ export const isLikeKind = (
   }
   return false
 }
+
+export const getTokenId = (
+  coreWallet: EdgeCurrencyWallet,
+  currencyCode: string
+): string | undefined => {
+  const { allTokens } = coreWallet.currencyConfig
+  return Object.keys(allTokens).find(
+    edgeToken => allTokens[edgeToken].currencyCode === currencyCode
+  )
+}
+
+export const consify = (val: any): void =>
```

Comment:
This name means something else in other programming circles. Also, we should *never* be using console.log in here, unless it's temporary debugging code. So maybe call it `prettyDebugLog` or something.

Reply:
this is for temporary debugging. I renamed to `prettyLogObject` since it's specifically for logging objects.

Reply:
renamed to prettyObjectLog and moved to utils.ts